linode/lke/resource.go

```diff
@@ -209,6 +216,12 @@ func updateResource(ctx context.Context, d *schema.ResourceData, meta interface{
 		}
 	}
 
+	for _, id := range updatedIds {
+		client.WaitForLKEClusterConditions(ctx, id, linodego.LKEClusterPollOptions{
```

Comment:
I was thinking about this earlier and I think it might actually be better to have a separate polling function to check if each node pool has at least one `ready` node (from the Linode API). It looks like the current implementation would pass automatically if any nodes that were previous provisioned remained since this function checks for only one `ready` node.

Fortunately it looks like we already have a function for doing this here 🎉 : https://github.com/linode/terraform-provider-linode/blob/dev/linode/lke/cluster.go#L181